edx_sysadmin/static/css/style.css

```diff
@@ -0,0 +1,69 @@
+/* 
```

Comment:
I think we should create this file in `edx_sysadmin/static/edx_sysadmin/css/style.css` so the path could never get ambiguous.

Reply:
I believe, It will always look for static files under `edx_sysadmin` in platform.

Reply:
I might need to jog my memory but as far as what I can remember, in dev environment static assets are directly fetched from `/static/` folders and they might not mingle with each other but in production all the assets are collected in a separate directory using `collectstatic` command and that command will not take into account that the assets are from which app. So two apps having the same named files might override each other.

In-short if we have a very generic name like `style.css` then after doing collectstatic they might not work as perfectly as they should. So having a name directory would be a good approach.